OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -67,6 +67,35 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						break;
 					}
 
+				case "ReplacePlayerWithBot":
+					{
+						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
+						if (client != null && world != null)
+						{
+							var player = world.FindPlayerByClient(client);
+							if (player == null || player.IsBot)
+								break;
+
+							client.State = Session.ClientState.Ready;
+
+							var admin = orderManager.LobbyInfo.Clients.FirstOrDefault(c => c.IsAdmin);
```

Comment:
@pchote Then this won't fire. See https://github.com/OpenRA/OpenRA/pull/10502#discussion-diff-49606608R583

@Phrohdoh See https://github.com/OpenRA/OpenRA/pull/10502/files#diff-e34ac0a837f43698d67024b60cd1cf12R598 but I doubt "ping" would be a good measurement.